clusterloader2/pkg/framework/config/client_config.go

```diff
@@ -107,7 +107,7 @@ func transportHack(config *restclient.Config) error {
 		}).DialContext,
 	})
 	config.WrapTransport = transportConfig.WrapTransport
-	config.Dial = transportConfig.Dial
+	// config.Dial = transportConfig.Dial // TODO: What replaces this?
```

Comment:
I wasn't sure how to handle this new struct mismatch.

Reply:
https://github.com/kubernetes/kubernetes/commit/3313a70d5bcc40a39f99f482c18effc9de6072ba shows what to do here